docs/source/multidc/multidc.md

```diff
@@ -586,12 +586,12 @@ auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqns
 
 Save the output, replace the token with your own, and patch the secret in the second datacenter with the below command:
 ```shell
-kubectl --context="${CONTEXT_DC2}"-n=scylla patch secret/scylla-cluster-auth-token --type='json' -p='[{"op": "replace", "path": "/data/auth-token.yaml", "value": "auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf"}]'
+kubectl --context="${CONTEXT_DC2}" -n=scylla patch secret/scylla-cluster-auth-token--type='json' -p='[{"op": "add", "path": "/stringData", "value": {"auth-token.yaml": "auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf"}}]'
```

Comment:
`"path": "/stringData"` - can this be done with a specific key - it seems quite bug prone to possibly erase other keys this way

Reply:
I'm afraid not with type=json
```shell
$ kubectl -n=scylla patch secret/scylla-auth-token --type='json' -p='[{"op": "add", "path": "/stringData/auth-token.yaml", "value": "auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf"}]'
error: add operation does not apply: doc is missing path: "/stringData/auth-token.yaml": missing value
```

Possible with type=merge though:
```shell
kubectl -n=scylla patch secret/scylla-auth-token --type='merge' -p='{"stringData": {"auth-token.yaml": "auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf"}}'
```

Reply:
But "path": "/stringData" does not erase the existing keys

Reply:
I've missed you've changed the replace op to add. With stringData being always empty it will eventually do a merge when it internally converts StringData to data.

It's a bit neat and weird at the same time :) @zimnx do you have a preference between `data`, `stringData`?
/assign @zimnx

Reply:
stringData is in my opinion cleaner as we are using it for a reason it was added.